commands/commands.go

```diff
@@ -380,6 +380,12 @@ var Commands = []cli.Command{
 		Action:      cmdStart,
 	},
 	{
+		Name:        "status",
+		Usage:       "Get the status of a machine",
+		Description: "Argument(s) are one or more machine names.",
```

Comment:
This implies it can handle multiple machines, but the purpose of the command is to report on _one_ machine.